go/ct/spc/specification_test.go

```diff
@@ -184,7 +184,9 @@ func TestSpecification_AtMostOneCodeAtPC(t *testing.T) {
 	}
 }
 
-func TestSpecification_NumberOfTestCasesMatchesRuleInfo(t *testing.T) {
+// TODO: re-enable this test when runtime is not an issue anymore.
+// note: before commiting run it locally where timeout is not an issue.
+func Specification_NumberOfTestCasesMatchesRuleInfo(t *testing.T) {
```

Comment:
Is this a merge error or is this test deliberately disabled again?

Reply:
deliberately disabled again.